Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,3 +1,23 @@
-.small-card {
-  width: 50%;
+@import 'media-queries';
+
+.mat-card{
+  display: flex;
+  flex-direction: column;
+  flex: 1;
 }
+
+// mat-card-footer doesn't align with bottom, you have to user manually styling
+// --> https://github.com/angular/components/issues/11094
+.mat-card-content{
+  flex-grow: 1;
+  overflow: auto;
+}
+// manually styling ends here
```

Comment:
If you can't use classes from `styles.scss` this looks okay for me.

Reply:
Where did someone talked about removing the styles in `styles.scss`?

Reply:
I put it now to the overrides.scss